data/privacy_policy.html

```diff
@@ -0,0 +1,306 @@
+  improve the service.
+</p>
+
+<h2 class="govuk-heading-l">Why our use of your personal data is lawful</h2>
+<p>
+  In order for our use of your personal data to be lawful, we need to meet one, or more,
+  conditions in the data protection legislation.
+</p>
+<p>
+  DfE processes your data under the basis of <strong>Public Task (Article 6(1)(e) of GDPR)</strong>. In this
+  case, it is necessary for the department to carry out this work, and to use the information for
+  these purposes, to exercise its functions.
+</p>
+
+<h2 class="govuk-heading-l">What personal data we collect</h2>
+<p>
+  The personal data we collect will depend on whether you are a candidate, school induction
+  tutor, teacher training provider or induction provider.
+</p>
+
+<h3 class="govuk-heading-m">What personal data we collect</h3>
+<p>
+  If you’re a candidate using the Manage Continuing Professional Development Service we
+  will collect the following information about you:
+</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>your name</li>
+  <li>your phone number</li>
+  <li>your email address</li>
+  <li>your teacher reference number (TRN)</li>
+  <li>your date of birth</li>
+  <li>your national insurance number</li>
+  <li>where you work</li>
+  <li>what your role is</li>
+</ul>
+
+<h3 class="govuk-heading-m">What data we collect if you are a school induction tutor</h3>
+<p>
+  If you are a school induction tutor using the Manage Continuing Professional Development
+  Service we will collect the following information about you:
+</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>your name</li>
+  <li>your email address</li>
+  <li>where you work</li>
+  <li>what your role is</li>
+</ul>
+
+<h3 class="govuk-heading-m">What data we collect if you work for a teacher training provider or induction provider</h3>
+<p>
+  If you process teacher training applications or ECF induction registrations we will collect the
+  following information about you:
+</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>your name</li>
+  <li>your email address</li>
+  <li>your phone number</li>
+  <li>where you work</li>
+  <li>what your role is</li>
+</ul>
+
+<h2 class="govuk-heading-l">How we use your data</h2>
+<h3 class="govuk-heading-m">Processing applications and registrations</h3>
+<p>
+  Processing applications and registrations includes the following:
+</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>sending your application or registration details to teacher training providers and ECF</li>
+  <li>induction providers</li>
+  <li>validating teacher reference numbers (TRN)</li>
+  <li>working out any funding which candidates are entitled to</li>
+  <li>getting in touch if there’s a security issue concerning your data</li>
+  <li>getting in touch with candidates, teacher training providers and ECF induction</li>
+  <li>oviders about application and registrations</li>
```

Comment:
Typo!

Reply:
Can't even copy-paste correctly... Actually, li on lines 120 and 121 should be just a single li :D